m3u8/model.py

```diff
@@ -1119,6 +1137,57 @@ def dumps(self):
     def __str__(self):
         return self.dumps()
 
+class DateRangeList(TagList):
+    pass
+
+class DateRange(object):
+    def __init__(self, **kwargs):
+        self.id = kwargs.get('id')
```

Comment:
Maybe id and start_date can use the `kwargs['id']` syntax since they are required. What do you think?

Reply:
I will change id, but can't change start_date for the reasons described at https://github.com/globocom/m3u8/pull/198#discussion_r393658553

Reply:
Fixed via 22bce1e